.travis.yml

```diff
@@ -7,12 +7,17 @@ addons:
     - rpm
 
 language: python
-python:
-  - "2.7"
-  - "3.3"
+matrix:
+  include:
+    - python: 2.7
+      env: TOX_ENV=py27
+    - python: 3.3
+      env: TOX_ENV=py33
+    - python: 3.4
+      env: TOX_ENV=py34
+    - python: 3.5
+      env: TOX_ENV=py35
```

Comment:
This ran on 3.3 only because I never bothered to update it. I guess it can run only on 2.7 and 3.5 unless there is some specific reason to do otherwise.